internal/fs/wrappers/monitoring.go

```diff
@@ -60,8 +85,10 @@ func fsErrStr(err error) string {
 	return DefaultFSError.Error()
 }
 
-func recordOp(ctx context.Context, method string, fsErr error) {
+// recordOp records file system operation count, operation error count and the operation latency
```

Comment:
every comment should start with capital letter and end with fullstop. Please change everwhere